CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,37 @@
 # Code of Conduct
 
-The [Node.js Code of Conduct][] applies to this team.
+The OpenJS Foundation and its member projects use the Contributor
+Covenant v1.4.1 as its Code of Conduct. Refer to the following
+for the full text:
+
+* [english](https://www.contributor-covenant.org/version/1/4/code-of-conduct)
+* [translations](https://www.contributor-covenant.org/translations)
+
+Refer to the section on reporting and escalation in this document for the specific emails that can be used to report and escalate issues.
+
+# Reporting
+
+## Project Spaces
+
+For reporting issues in spaces related to a member project please use the email provided by the project for reporting. Projects handle CoC issues related to the spaces that they maintain.  Projects maintainers commit to:
+
+* maintain the confidentiality with regard to the reporter of an incident
+* to participate in the path for escalation as outlined in
+  the section on Escalation when required.
+
+## Foundation Spaces
+For reporting issues in spaces managed by the OpenJS Foundation, for example, repositories within the OpenJS organization, use the email `report@openjsf.org`. The CrossProjectCouncil (CPC) is responsible for managing these reports and commits to:
```

Comment:
`CrossProjectCouncil` reads funny to me. Should it be one word? I prefer simply `Cross Project Council`

Reply:
nope, not sure why it ended up that way, fixing.